contributor/es/02-becoming-a-contributor-article-es.asciidoc

```diff
@@ -0,0 +1,38 @@
+
+Sí, habrá ocasiones en las que necesites urgentemente la corrección de un error en tu línea de tiempo que no tiene la misma prioridad en el equipo anfitrión. Convertirte en Colaborador te permite actuar por tí mismo y apoyar al equipo anfitrión en la corrección de ese fallo.
+
+Esta forma de trabajar requiere un cambio de mentalidad para muchos: en lugar de esperar a que se implementen las características o se arreglen los errores, en lugar de trabajar alrededor de los problemas, ahora hay una tercera vía. Dedica tu tiempo y energía a comprobar con el proyecto InnerSource cuáles son tus necesidades, y luego realiza el cambio directamente en el proyecto compartido. Así que, además de tus habilidades de codificación, necesitas habilidades de comunicación para tener éxito en un proyecto InnerSource - para articular claramente tus necesidades y llegar a una solución que funcione tanto para tu equipo como para el equipo anfitrión.
+
+"Pero podría simplemente ir y bifurcar el proyecto, hacer mis cambios allí y ahorrarme toda esta sobrecarga de coordinación". Claro, bifurcar el proyecto es una forma de hacer tu trabajo. Pero, a largo plazo, esto significa que tienes que mantener esa versión bifurcada para tu equipo, y llevar tus cambios a cualquier nueva versión que haga el equipo anfitrión. Contribuir con tus cambios al equipo anfitrión también significa que te beneficias de su profundo conocimiento del componente. Pueden detectar problemas en tu parche que, de otro modo, sólo serían obvios en producción.
+
+Un buen colaborador puede decidir cómodamente cuándo tiene sentido, tanto desde el punto de vista técnico como empresarial, introducir una dependencia y reutilizar un componente en lugar de duplicar el trabajo. Pueden hablar con la dirección para explicar los beneficios de las contribuciones de InnerSource.
+
+=== Alcance de las contribuciones de InnerSource
+
+¿Así que Inner__Source__ es sólo sobre __Código __Fuente? Por supuesto que no. Si el negocio de tu equipo depende de un componente externo, quieres asegurarte de que está bien mantenido y funciona bien. Como colaborador de InnerSource, puedes ayudar al equipo anfitrión de múltiples maneras. Informar de los problemas que ve al utilizar el componente es una valiosa contribución. Crear o arreglar casos de prueba que muestren que el código no funciona como se espera es valioso. También lo es mejorar la documentación, para que otros pasen menos tiempo usándola y contribuyendo a ella. Dar soporte a otros usuarios o ayudar con la clasificación y priorización de errores puede ser una contribución valiosa. Mejorar las compilaciones es otro ejemplo de contribución valiosa.
```

Comment:
- The space should be outside the word.
- Maybe _**múltiples**_ is the way you intend it to be on the text, as it's probably more formal. I'm more used to hear _**varias**_ or _**diversas**_ though.

```suggestion
¿Así que Inner__Source__ es sólo sobre __Código__ Fuente? Por supuesto que no. Si el negocio de tu equipo depende de un componente externo, quieres asegurarte de que está bien mantenido y funciona bien. Como colaborador de InnerSource, puedes ayudar al equipo anfitrión de varias maneras. Informar de los problemas que ve al utilizar el componente es una valiosa contribución. Crear o arreglar casos de prueba que muestren que el código no funciona como se espera es valioso. También lo es mejorar la documentación, para que otros pasen menos tiempo usándola y contribuyendo a ella. Dar soporte a otros usuarios o ayudar con la clasificación y priorización de errores puede ser una contribución valiosa. Mejorar las compilaciones es otro ejemplo de contribución valiosa.
```